src/citrine/_session.py

```diff
@@ -303,6 +304,10 @@ def checked_get(self, path: str, **kwargs) -> Response:
         """Execute a GET request to a URL and utilize error filtering on the response."""
         return self.checked_request('GET', path, **kwargs)
 
+    def get_team_id_from_project_id(self, *, project_id: UUID) -> UUID:
```

Comment:
I am missing why we do not simply make a @property that is project.team that returns a Team object